downstream/modules/platform/proc-controller-route-options.adoc

```diff
@@ -7,5 +7,6 @@ The {PlatformName} operator installation form allows you to further configure yo
 . Click *Advanced configuration*.
 . Under *Ingress type*, click the drop-down menu and select *Route*.
 . Under *Route DNS host*, enter a common host name that the route answers to.
-. Under *Route TLS termination mechanism*, click the drop-down menu and select *Edge* or *Passthrough*.
+. Under *Route TLS termination mechanism*, click the drop-down menu and select *Edge* or *Passthrough*. termiination. For most instances *Edge* should be selected.
```

Comment:
```suggestion
. Under *Route TLS termination mechanism*, click the drop-down menu and select *Edge* or *Passthrough*. For most instances *Edge* should be selected.
```
Looks like "termination." was a typo or something that should have been deleted.